src/native/corehost/fxr/sdk_resolver.cpp

```diff
@@ -519,17 +614,16 @@ bool sdk_resolver::resolve_sdk_path_and_version(const pal::string_t& dir, pal::s
                 resolved_version_str.empty() ? _X("none") : resolved_version_str.c_str()
             );
 
-            changed = true;
+            found = true;
             resolved_version = version;
             resolved_version_str = std::move(version_str);
         });
 
-    if (changed)
+    if (found)
     {
         sdk_path = dir;
         append_path(&sdk_path, resolved_version_str.c_str());
     }
 
-    // Not yet fully resolved
-    return false;
+    return found;
```

Comment:
This is fine to change because SDK multi-level lookup is no longer a thing. Previously, `resolve_sdk_path_and_version` would only return `true` if an exact match was found, such that we would keep going through other root paths (from multi-level lookup) to see if there is a better match. Now, it returns whether or not a match was found in the specified `dir`. With the custom paths, the user specifies the exact search order - we want to take the first match.